keyboards/keebio/bfo9000/keymaps/via/via.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
This file has to be removed, as it lives in a [different repo](https://github.com/the-via/keyboards).